### PR TITLE
Added new module 

### PR DESCRIPTION
**Pull request by Tom-Brughmans**
I added a new module

### CODE REVIEW

**Comment by Tom-Brughmans** [comment]
First complete documentation of the module 'network structures', including NASSA.yml , references, image

**Comment by Tom-Brughmans** [comment]
Added module 2022-Brughmans-002 including tutorial .pdf, input .graphml file, screenshot, references.
Made small changes to 2022-Brughmans-001 by including tutorial .pdf, edits to readme, and linking to module 002 in .yml

File: 2022-Brughmans-001/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) <YEAR> <COPYRIGHT HOLDER>
```
Comment (driesdaems10) [inline 2022-Brughmans-001/LICENSE:3]: Please add year and name

File: 2022-Brughmans-001/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) <2022> <COPYRIGHT HOLDER>
```
Comment (driesdaems10) [inline 2022-Brughmans-001/LICENSE:3]: Add your name!

File: 2022-Brughmans-001/LICENSE (hunk 1 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-001/LICENSE]: year added

File: 2022-Brughmans-001/NASSA.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+id: 2022-Brughmans-001
+nassaVersion: 0.3.0
+moduleType: Submodel
+title: Network structures
+moduleVersion: 1.0.0
+contributors:
+ - name: Brughmans, Tom (with no accent marks, first upper-case letters, and surname matching the id and root directory name)
```
Comment (driesdaems10) [inline 2022-Brughmans-001/NASSA.yml:7]: Delete template text between parentheses

File: 2022-Brughmans-001/NASSA.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+   roles: [ "Author", "Compiler", "Contributor", "Copyright Holder", "Creator"]
+   email: t.b@cas.au.dk
+   orcid: 0000-0002-1589-7768
+lastUpdateDate: 2022-12-02
+description: >
+  A draft model with some useful code for creating different network structures using the Netlogo NW extension: small-world, preferential attachment, circular, star, wheel, lattice, random, nearest neighbours. This model is used for the following tutorial: Brughmans, T. (2018). Network structures and assembling code in Netlogo, Tutorial, https://archaeologicalnetworks.wordpress.com/resources/#structures .
+relatedModules: [ 2022-Brughmans-001 ]
```
Comment (driesdaems10) [inline 2022-Brughmans-001/NASSA.yml:14]: Delete self-citation

File: 2022-Brughmans-001/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+This version created 01/09/2018
+Netlogo version used: 6.0.1
+Extension used: nw (pre-packaged with Netlogo 6.0.1)
+https://ccl.northwestern.edu/netlogo/6.0-BETA1/docs/nw.html 
+Cite this tutorial as:
+Brughmans, T. (2018). Network structures and assembling code in Netlogo, Tutorial, https://archaeologicalnetworks.wordpress.com/resources/#structures  .
+
+This tutorial provides an introduction to finding and assembling pre-existing code to quickly create complex models. It uses code and data linked to in the https://projectmercury.eu pages. We will create a Roman transport network by reusing existing code that draws on the open access ORBIS dataset (http://orbis.stanford.edu/), we will create alternative network structures by reusing existing code, and we will explore the impact these different network structures have in light of simple economic processes. This tutorial will reveal the importance of not reinventing the wheel, of searching for appropriate existing code and letting your model building be inspired by others’ previous work.
+
+Tutorial document availabe as a PDF in the [netlogo_implementtion folder](netlogo_implementation/Netlogo_network-structures_v0.1.pdf)
```
Comment (driesdaems10) [inline 2022-Brughmans-001/README.md:18]: Move this up, above 'cite tutorial as'

File: 2022-Brughmans-001/documentation/designDetails.md [hunk 6]
```diff
@@ -0,0 +1 @@
+# Design details
```
Comment (driesdaems10) [inline 2022-Brughmans-001/documentation/designDetails.md:1]: Delete unused files (see elsewhere as well)

File: 2022-Brughmans-001/netlogo_implementation/2022-Brughmans-001.nlogo [hunk 7]
```diff
@@ -0,0 +1,719 @@
+extensions [nw]
+
+;;; PROCEDURES FOR CREATING A NEW NETWORK STRUCTURE WITH A NEw SET OF NODES
```
Comment (driesdaems10) [inline 2022-Brughmans-001/netlogo_implementation/2022-Brughmans-001.nlogo:3]: NEW instead of NEw

**Review by driesdaems10 — CHANGES_REQUESTED** [review]
Thank you for your submission, Tom. This looks like a really useful addition to our library. Please find a few humble suggestions for improvement.

File: 2022-Brughmans-002/LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) <YEAR> <COPYRIGHT HOLDER>
```
Comment (MarekVlach) [inline 2022-Brughmans-002/LICENSE:3]: Please, put the proper info.

File: 2022-Brughmans-002/NASSA.yml [hunk 9]
```diff
@@ -0,0 +1,58 @@
+id: 2022-Brughmans-002
+nassaVersion: 0.3.0
+moduleType: Submodel
+title: Importing a Roman Transport network
+moduleVersion: 1.0.0
+contributors:
+ - name: Brughmans, Tom (with no accent marks, first upper-case letters, and surname matching the id and root directory name)
```
Comment (MarekVlach) [inline 2022-Brughmans-002/NASSA.yml:7]: Delete automated text

File: 2022-Brughmans-002/NASSA.yml [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+   roles: [ "Author", "Compiler", "Contributor", "Copyright Holder", "Creator"]
```
Comment (MarekVlach) [inline 2022-Brughmans-002/NASSA.yml:8]: Choose the role

File: 2022-Brughmans-002/README.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+Netlogo version used: 6.0.1
+Extension used: nw (pre-packaged with Netlogo 6.0.1)
+https://ccl.northwestern.edu/netlogo/6.0-BETA1/docs/nw.html 
+Cite this tutorial as:
+Brughmans, T. (2018). Importing a Roman Transport network with Netlogo, Tutorial, https://archaeologicalnetworks.wordpress.com/resources/#transport  .
+
+
+This tutorial provides an introduction to finding and assembling pre-existing code to quickly create complex models. It uses code and data linked to in the https://projectmercury.eu pages. We will create a Roman transport network by reusing existing code that draws on the open access ORBIS dataset (http://orbis.stanford.edu/), we will create alternative network structures by reusing existing code, and we will explore the impact these different network structures have in light of simple economic processes. This tutorial will reveal the importance of not reinventing the wheel, of searching for appropriate existing code and letting your model building be inspired by others’ previous work.
+
+Tutorial document availabe as a PDF in the [netlogo_implementtion folder](netlogo_implementation/Netlogo_Roman-transport_v0.1.pdf)
```
Comment (MarekVlach) [inline 2022-Brughmans-002/README.md:19]: Please, put that upwards

File: 2022-Brughmans-002/documentation/designDetails.md [hunk 12]
```diff
@@ -0,0 +1 @@
+# Design details
```
Comment (MarekVlach) [inline 2022-Brughmans-002/documentation/designDetails.md:1]: Delete the file

File: 2022-Brughmans-002/documentation/tableOfContents.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
+Module name or title
```
Comment (MarekVlach) [inline 2022-Brughmans-002/documentation/tableOfContents.md:1]: Delete the file

File: 2022-Brughmans-002/netlogo_implementation/documentation/instructions.md [hunk 14]
```diff
@@ -0,0 +1 @@
+# Instructions
```
Comment (MarekVlach) [inline 2022-Brughmans-002/netlogo_implementation/documentation/instructions.md:1]: Fill in or delete

File: 2022-Brughmans-002/netlogo_implementation/documentation/tableOfContents.md [hunk 15]
```diff
@@ -0,0 +1,13 @@
+Module name or title
```
Comment (MarekVlach) [inline 2022-Brughmans-002/netlogo_implementation/documentation/tableOfContents.md:1]: Fill in or delete

**Review by MarekVlach — COMMENTED** [review]
Thank you for your contribution

File: 2022-Brughmans-001/NASSA.yml (hunk 4 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-001/NASSA.yml]: done

File: 2022-Brughmans-001/README.md (hunk 5 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-001/README.md]: done

File: 2022-Brughmans-001/documentation/designDetails.md (hunk 6 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-001/documentation/designDetails.md]: done

File: 2022-Brughmans-001/netlogo_implementation/2022-Brughmans-001.nlogo (hunk 7 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-001/netlogo_implementation/2022-Brughmans-001.nlogo]: done

File: 2022-Brughmans-002/LICENSE (hunk 8 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/LICENSE]: done

File: 2022-Brughmans-002/NASSA.yml (hunk 9 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/NASSA.yml]: done

File: 2022-Brughmans-002/README.md (hunk 11 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/README.md]: done

File: 2022-Brughmans-002/documentation/tableOfContents.md (hunk 13 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/documentation/tableOfContents.md]: done

File: 2022-Brughmans-002/documentation/designDetails.md (hunk 12 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/documentation/designDetails.md]: done

File: 2022-Brughmans-002/netlogo_implementation/documentation/tableOfContents.md (hunk 15 above)
Comment (Tom-Brughmans) [reply 2022-Brughmans-002/netlogo_implementation/documentation/tableOfContents.md]: done

**Pull request merged**